jarviscli/plugins/teachme.py

```diff
@@ -0,0 +1,420 @@
+from nltk.tokenize import word_tokenize as wt, sent_tokenize as st
+from nltk.corpus import wordnet
+from nltk.stem import WordNetLemmatizer
+from collections import Counter
+import nltk
+import numpy as np
+import matplotlib.pyplot as plt
+from nltk.corpus import stopwords
+from nltk import FreqDist
+import os
+import requests
+import json
+import spacy

```

Comment:
Add spacy, google_trans_new to installer/requirements.txt
Note @ me: Maybe migrate google translator plugin to google_trans_new?